content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst

```diff
@@ -1,161 +1,208 @@
 
-.. image:: third_party_shipper/third_party_shipper07.png
+.. image:: third_party_shipper/production.png
    :align: center
+   :alt: Show the "Test Environment" smart button.
 
-Company configuration
----------------------
-
-In order to compute the right price, the provider needs your company
-information. Be sure your address and phone number are correctly
-encoded.
+.. note::
+   For databases with the *Website* app installed, select the :guilabel:`Unpublished` smart button
+   to enable the delivery method on the website.
 
-.. image:: third_party_shipper/third_party_shipper03.png
-   :align: center
+   .. image:: third_party_shipper/published.png
+      :align: center
+      :alt: Display "Unpublished" smart button at the top of the delivery method page.
 
-To check your information, go to the **Settings** application and click on
-**General Settings**. Click on the first link **Configure your company data**.
+.. _inventory/shipping_receiving/configure-source-address:
 
-Product configuration
----------------------
+Source address
+--------------
 
-The shipper companies need the weight of your product, otherwise the
-price computation cannot be done.
+Configure the source address for package shipments by navigating to :menuselection:`Inventory app
+--> Configuration --> Warehouses` and selecting the desired warehouse.
 
-Go the menu :menuselection:`Sales --> Sales --> Products`. Open the products you
-want to ship and set a weight on it.
+On the warehouse configuration page, hover over the :guilabel:`Address` field and select the
+:guilabel:`➡️ (right arrow)` icon. Doing so opens the warehouse contact page.
 
-.. image:: third_party_shipper/third_party_shipper09.png
+.. image:: third_party_shipper/internal-link.png
    :align: center
+   :alt: Show the internal link icon that appears when hovering over "Address".
 
-.. tip::
-    The weight on the product form is expressed in kilograms.
-    Don't forget to make the conversion if you are used to the imperial
-    measurement system.
+Ensure the warehouse's :guilabel:`Address` and :guilabel:`Phone` number are entered accurately, as
+they are used to calculate cost of shipping.
 
-Sale process
-============
+.. image:: third_party_shipper/company.png
+   :align: center
+   :alt: Show company address and phone number.
 
-Sale order
-----------
+.. _inventory/shipping_receiving/configure-weight:
 
-.. image:: third_party_shipper/third_party_shipper05.png
-   :align: center
+Product weight
+--------------
 
-You can now choose the carrier on your sale order. Click on **Delivery
-method** to choose the right one.
+To calculate the price of shipping, specify the weight of products by going to
+:menuselection:`Inventory app --> Products --> Products`, and selecting the desired product.
 
-The price is computed when you save the sale order or when you click on
-**Set price**.
+Then, switch to the :guilabel:`Inventory` tab, define the :guilabel:`Weight` of the product, in
+kilograms.
```

Comment:
Why kilograms? Does it have to be? Can different products have different UoMs?

Reply:
You're right! I confirmed with an expert, and this field has become really flexible since 8 years ago, when the line was written (: